integration-tests/integration_tests/helpers/methods.py

```diff
@@ -18,7 +18,7 @@ def get_asid():
         asid_file = str(Path(ROOT_DIR) / "integration_tests/data/certs/asid.txt")
         asid = FileUtilities.get_file_string(asid_file)
     except:
-        asid = None
+        asid = 918999199084
```

Comment:
Why do we need to hardcode an ASID here?

Reply:
Accidentally committed it for my local integration tests